apps/files/src/components/FileDetails.vue

```diff
@@ -85,7 +85,11 @@ export default {
     },
 
     currentTab() {
-      return this.currentSidebarTab?.tab || this.defaultTab
+      if (this.currentSidebarTab) {
```

Comment:
What about `if(this.currentSidebarTab && this.currentSidebarTab.tab) {` here? Feels a bit weird to have to fallbacks to the same variable in place. ;-)